proxy-init/integration_test/iptables/http_test.go

```diff
@@ -180,8 +180,14 @@ func makeCallFromContainerToAnother(t *testing.T, fromPodNamed string, fromConta
 	return expectSuccessfulGetRequestToUrl(t, targetUrl)
 }
 
-func marksParallelIfConfigured(t *testing.T) {
-	t.Parallel()
+func checkIfIntegrationTestsAreEnabled(t *testing.T) {
+	integrationTestsEnvironmentVariable := "CONDUIT_INTEGRATION_TESTS_ENABLED"
```

Comment:
TIOLI -- you could move this up with the rest of the consts at the top of this file.